projects/train/train/data/supervised/supervised.py

```diff
@@ -52,10 +52,7 @@ def augment(self, X):
         snrs = self.snr_sampler.sample((N,)).to(X.device)
         responses = self.projector(*params, snrs, psds[mask], **polarizations)
         kernels = sample_kernels(
-            responses,
-            kernel_size=X.size(-1),
-            max_center_offset=self.pad_size,
-            coincident=True,
+            responses, kernel_size=X.size(-1), coincident=True
```

Comment:
Okay so now we have padded the waveforms such that sampling any window of `kernel_size` will satisfy our requested `right_pad` / `left_pad` constraints? This is way nicer and convinces me we should get rid of this `max_center_offset` `kwarg` all together.